backend/resources/hacknight.py

```diff
@@ -54,3 +54,38 @@ def get(self, id):
         return {'hacknights': hacknight_schema.dump(Hacknight.query.get_or_404(
             id))
         }, HTTPStatus.OK
+
+    @jwt_required
+    def patch(self, id):
+        hacknight = Hacknight.query.get_or_404(id)
+        participants = [
+            participant.id for participant in hacknight.participants
+        ]
+
+        json_data = request.get_json(force=True)
+        ids_schema = Schema.from_dict(
+            {"participants_ids": fields.List(fields.Int())}
+        )
+        try:
+            data = ids_schema().load(json_data)
+        except ValidationError as err:
+            return (err.messages), HTTPStatus.BAD_REQUEST
+
+        new_participants = [
+            id for id in data['participants_ids'] if id not in participants
+        ]
+
+        if not new_participants:
+            return {'message': 'No new participant has been provided'}, \
+                HTTPStatus.BAD_REQUEST
+
+        for new_participant in new_participants:
+            hacknight.participants.append(
+                Participant.query.get_or_404(new_participant)
+            )
+        db.session.add(hacknight)
+        db.session.commit()
+
+        hacknight_schema = HacknightSchema()
+        return {"hacknight": hacknight_schema.dump(hacknight)}, \
```

Comment:
I think we can remove `"hacknight"`.

Reply:
No reason for `hacknight` as a key here, we should return it directly.